README.md

```diff
@@ -4,7 +4,7 @@
 
 Aims to centralize all the technical knowledge of the 'Kingdom Hearts' game series in one place, providing documentation, tools, code libraries, and the foundation for modding the commercial games.
 
-[![Build Status](https://dev.azure.com/xeeynamo/OpenKH/_apis/build/status/Xeeynamo.OpenKh?branchName=master) ![Tests](https://img.shields.io/azure-devops/tests/xeeynamo/OpenKh/4) ![Coverage](https://img.shields.io/azure-devops/coverage/xeeynamo/OpenKh/4)](https://dev.azure.com/xeeynamo/OpenKH/_build/latest?definitionId=4&branchName=master)
+[![Build Status](https://dev.azure.com/Xeeynamo/OpenKH/_apis/build/status/Xeeynamo.OpenKh?branchName=master) ![Tests](https://img.shields.io/azure-devops/tests/xeeynamo/OpenKh/4) ![Coverage](https://img.shields.io/azure-devops/coverage/xeeynamo/OpenKh/4)](https://dev.azure.com/xeeynamo/OpenKH/_build/latest?definitionId=4&branchName=master)
```

Comment:
I suggest to replace Azure Pipeline's build badge with GitHub Actions can produce build badge.

[Adding a workflow status badge - GitHub Docs](https://docs.github.com/en/actions/monitoring-and-troubleshooting-workflows/adding-a-workflow-status-badge)

This is alternative one:

```md
[![Build Status](https://github.com/OpenKh/OpenKh/actions/workflows/dotnet.yml/badge.svg)](https://github.com/OpenKH/OpenKh/actions/workflows/dotnet.yml)
```

---

Badge image url:
https://github.com/OpenKh/OpenKh/actions/workflows/dotnet.yml/badge.svg

Badge image:
![](https://github.com/OpenKh/OpenKh/actions/workflows/dotnet.yml/badge.svg)

Composed badge:
[![Build Status](https://github.com/OpenKh/OpenKh/actions/workflows/dotnet.yml/badge.svg)](https://github.com/OpenKH/OpenKh/actions/workflows/dotnet.yml)